### ISSUE TITLE
Long telomere length

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10260]
**Preferred term label:**
Long telomere length

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C3178816

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0031412

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/615848

**Your nano-attribution (ORCID)**
